armory/__main__.py

```diff
@@ -248,16 +251,23 @@ def run(command_args, prog, description):
     coloredlogs.install(level=args.log_level)
 
     try:
-        config = load_config(args.filepath)
+        if args.filepath == "-":
+            logger.info("Reading config from stdin...")
+            config = load_config_stdin()
+        else:
+            config = load_config(args.filepath)
     except ValidationError as e:
         logger.error(
             f"Could not validate config: {e.message} @ {'.'.join(e.absolute_path)}"
         )
         sys.exit(1)
     except json.decoder.JSONDecodeError:
-        logger.exception(f"Could not decode {args.filepath} as a json file.")
-        if not args.filepath.lower().endswith(".json"):
-            logger.warning(f"{args.filepath} is not a '*.json' file")
+        if args.filepath == "-":
+            logger.exception("'stdin' did not provide a json-parsable input")
```

Comment:
The traceback from this `logging.exception()` (and the other on line 268 below) obscures the error message and does not provide much useful feedback to the user.

Could this (and L268) be instead `logging.error()`